release notes/v1.0.0-rc1.md

```diff
@@ -0,0 +1,186 @@
+
+## New features
+
+### Browser: Tracking network requests and responses [#4290](https://github.com/grafana/k6/issues/4290), [#4296](https://github.com/grafana/k6/issues/4296)
+
+The [browser module](https://grafana.com/docs/k6/latest/javascript-api/k6-browser/) adds support for tracking network requests and responses. This feature is especially useful for validating certain aspects of the requests and responses to determine whether the test was successful. It can also be used to debug issues with the test script or the tested application. See the [documentation](https://grafana.com/docs/k6/latest/javascript-api/k6-browser/page/on/) for more information.
+
+For example, to track all requests and responses made by a page, you can use the following script:
+
+```javascript
+import { browser } from 'k6/browser';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+          type: 'chromium',
+        },
+      },
+    },
+  },
+};
+
+export default async function () {
+  const page = await browser.newPage();
+
+  // registers a handler that logs all requests made by the page
+  page.on('request', async request => console.log(request.url()));
+  // registers a handler that logs all responses received by the page
+  page.on('response', async response => console.log(response.url()));
+
+  await page.goto('https://quickpizza.grafana.com/', { waitUntil: 'networkidle' });
+  await page.close();
+}
+```
+
+Output might look like this:
+
+```bash
+INFO[0000] https://quickpizza.grafana.com/                  source=console
+INFO[0001] https://quickpizza.grafana.com/api/tools         source=console
+INFO[0001] https://quickpizza.grafana.com/images/pizza.png  source=console
+...
+```
+
+### `k6/experimental/webcrypto` promoted to stable and available globally under `crypto` [#4278](https://github.com/grafana/k6/pull/4278)
+
+With this release, the `k6/experimental/webcrypto` module is promoted to stable and available globally under `crypto`. So you don't need to import it anymore.
+
+```javascript
+export default function () {
+  const myUUID = crypto.randomUUID();
+
+  console.log(myUUID);
+}
+```
+
+`k6/experimental/webcrypto` is deprecated and will be removed in `v1.1.0`.
+
+### Support for custom templates in `k6 new` command [#4618](https://github.com/grafana/k6/pull/4618)
+
+The `k6 new` command now accepts a path to a file to use as a template for the new script. Templates use [Go templates syntax](https://pkg.go.dev/text/template#pkg-overview) and can include the following variables:
+
+- `ScriptName`: The name of the new script.
+- `ProjectID`: The ID of the Grafana Cloud project to use for the new script.
+
+To generate a new script using a custom template, use the following command:
+
+```sh
+k6 new --template /path/to/my-template.js
+```
+### Secret Sources [#4514](https://github.com/grafana/k6/pull/4514), [#4621](https://github.com/grafana/k6/pull/4621)
```

Comment:
If we have docs it's worth to put the link here

Reply:
https://github.com/grafana/k6-docs/pull/1905